src/CommunityToolkit.Maui/Layouts/DockLayout.shared.cs

```diff
@@ -21,10 +21,10 @@ public class DockLayout : Layout, IDockLayout
 	public static readonly BindableProperty VerticalSpacingProperty = BindableProperty.Create(nameof(VerticalSpacing), typeof(double), typeof(DockLayout), 0.0);
 
 	/// <inheritdoc/>
-	public bool ExpandLastChild
+	public bool IsLastChildFill
```

Comment:
I'm not a fluent english speaker, so I would like @brminnick to confirm if the new name is correct/makes sense

Reply:
Thanks. Renamed to `ShouldExpandLastChild` as per brminnick.